src/main/io/rc_controls.c

```diff
@@ -159,11 +160,16 @@ void processRcStickPositions(rxConfig_t *rxConfig, throttleStatus_e throttleStat
             // Disarming via ARM BOX
 
             if (ARMING_FLAG(ARMED) && rxIsReceivingSignal() && !failsafeIsActive()  ) {
-                if (disarm_kill_switch) {
-                    mwDisarm();
-                } else if (throttleStatus == THROTTLE_LOW) {
-                    mwDisarm();
+                rcDisarmTicks++;
```

Comment:
This counter counts as long as ARMED. It will just catch first three glitches.

Resetting counter in `if (IS_RC_MODE_ACTIVE(BOXARM)) {` will fix it